packages/falter-common/Makefile

```diff
@@ -41,4 +54,82 @@ define Package/falter-common/install
 	$(SED) 's,%R,$(FALTER_REVISION),g' $(1)/etc/freifunk_release
 endef
 
+
+define Package/falter-more
+	$(call Package/falter-standard/template)
+	TITLE:=Falter more files
+	EXTRA_DEPENDS:=falter-common, falter-profiles, luci-app-ffwizard-falter, falter-berlin-migration, falter-berlin-tunneldigger, falter-policyrouting, falter-berlin-ssid-changer,
+	EXTRA_DEPENDS+= falter-berlin-firewall-defaults, qos-scripts, firewall, luci-app-firewall, luci-i18n-firewall-de, luci-i18n-firewall-en,
+	EXTRA_DEPENDS+= falter-berlin-autoupdate, falter-berlin-autoupdate-keys, luci-app-falter-autoupdate, luci-i18n-falter-autoupdate-de,
+	EXTRA_DEPENDS+= falter-berlin-service-registrar, luci-app-falter-service-registrar, luci-i18n-falter-service-registrar-de,
+	EXTRA_DEPENDS+= collectd-mod-cpu, collectd-mod-load
+endef
+
+define Package/falter-more/description
+	Additional configs and scripts that enable participation in Falter-based Freifunk networks.
+endef
+
+define Package/falter-more/install
+	true
+endef
+
+
+define Package/falter-standard
+	$(call Package/falter-standard/template)
+	TITLE:=Falter Standard firmware (tunnel uplink)
+	EXTRA_DEPENDS:=falter-more, falter-berlin-uplink-tunnelberlin, wpad-mini
+endef
+
+define Package/falter-standard/description
+	Complete firmware for Freifunk networks, including OLSR meshing, Tunneldigger uplinks, Web UI, service hosting, and automatic updates.
+endef
+
+define Package/falter-standard/install
+	true
+endef
+
+
+define Package/falter-notunnel
+	$(call Package/falter-standard/template)
+	TITLE:=Falter No-Tunnel firmware (direct uplink)
+	EXTRA_DEPENDS:=falter-more, falter-berlin-uplink-notunnel
+endef
+
+define Package/falter-notunnel/description
+	Complete firmware for Freifunk networks, using direct uplink instead of Tunneldigger.
+endef
+
+define Package/falter-notunnel/install
+	true
+endef
+
+
+define Package/falter-backbone
+	$(call Package/falter-standard/template)
+	TITLE:=Falter Backbone firmware (mesh only)
+	EXTRA_DEPENDS:=falter-common, falter-berlin-tunneldigger
+endef
+
+define Package/falter-backbone/description
+	Firmware for Freifunk backbone routers, without uplink nor automatic updates.
+endef
+
+define Package/falter-backbone/install
+	true
+endef
+
+# XXX do via early uci-defaults instead (00-falter-skip-firewall)
+# XXX then merge firewall-defaults into falter-common
+define Package/falter-backbone/postinst
+#!/bin/sh
+f="$${IPKG_INSTROOT}/etc/config/defaults"
+echo "config packages 'default'" >  $f
+echo "  option firewall '1'"     >> $f
+endef
```

Comment:
firewall is the only *-defaults package that's not used by all three variants. I skipped it in the previous PR #384. If we take care to skip firewall config in the backbone variant, it can be merged into falter-common like the other *-defaults scripts.